docker-compose.yml

```diff
@@ -11,7 +11,7 @@ agenda:
     - mongodb
 
 mongodb:
-  image: mongo:latest
+  image: mongo:latest@sha256:3a09cd85fb4e76f1d5832f9ea1d4e7481f76e807389b7d8ea6ac4d4ba96f83e5
```

Comment:
Let's use `3.6` or `3.6.5` tag for clarity instead. https://hub.docker.com/_/mongo/